cmd/frontend/internal/httpapi/internal.go

```diff
@@ -170,6 +170,27 @@ func serveConfiguration(w http.ResponseWriter, r *http.Request) error {
 	return nil
 }
 
+type searchOptions struct {
+	LargeFiles []string
+}
+
+// serveSearchConfiguration is _only_ used by the zoekt index server. Zoekt does
+// not depend on frontend and therefore does not have access to `conf.Watch`.
+// Additionally, it only cares about certain search specific settings so this
+// search specific endpoint is used rather than serving the entire site settings
+// from /.internal/configuration.
+func serveSearchConfiguration(w http.ResponseWriter, r *http.Request) error {
+	largeFiles := conf.Get().SearchLargeFiles
+	opts := searchOptions{
+		LargeFiles: largeFiles,
+	}
+	err := json.NewEncoder(w).Encode(opts)
+	if err != nil {
+		return errors.Wrap(err, "Encode")
```

Comment:
nit: idiomatic go is to not capitalise error messages
```suggestion
		return errors.Wrap(err, "encode")
```